src/seedu/addressbook/ui/TextUi.java

```diff
@@ -124,46 +124,10 @@ public void showToUser(String... message) {
     public void showResultToUser(CommandResult result) {
         final Optional<List<? extends ReadOnlyPerson>> resultPersons = result.getRelevantPersons();
         if(resultPersons.isPresent()) {
-            showPersonListView(resultPersons.get());
+        	Formatter format = new Formatter(resultPersons.get());
+            format.showPersonListView();
```

Comment:
I don't think this line works. Remember to run tests to avoid regression errors.